src/app/map/[id]/context/ChoroplethContext.tsx

```diff
@@ -21,7 +21,12 @@ export const ChoroplethContext = createContext<{
   setBoundariesPanelOpen: () => null,
   selectedBivariateBucket: null,
   setSelectedBivariateBucket: () => null,
-  choroplethLayerConfig: getChoroplethLayerConfig(null, null, DEFAULT_ZOOM),
+  choroplethLayerConfig: getChoroplethLayerConfig(
```

Comment:
Optional
getChoroplethLayerConfig should really have a single object parameter from which arguments are destructured